app/views/buyers/accounts/_search_results.html.slim

```diff
@@ -15,7 +15,7 @@ table class="data" id="buyer_accounts"
 
       th class="actions"
         - if current_account.master?
-          = link_to 'Create', new_provider_admin_account_path, title: 'Create new tenant account', class: 'action add'
+          = link_to 'Create', new_provider_admin_account_path, title: 'Create new account', class: 'action add'
```

Comment:
so lets move the if, else inline as now the only difference is the path

Reply:
I agree 😄 It is simpler:
```slim
th class="actions"
  - if can?(:create, Account)
    = link_to 'Create', (current_account.master? ? new_provider_admin_account_path : new_admin_buyers_account_path), title: 'Create new account', class: 'action add'
```

And with this I am assuming that `can?(:create, Account)` for a master is always true, which should be, otherwise what we currently have here is already wrong 😄

Reply:
@Martouta @thomasmaas , I simplified it, can you check?